docs/view_components/progression_bar.rst

```diff
@@ -13,34 +15,40 @@ A Progression bar view-component has a number of advantages:
 Attributes
 ^^^^^^^^^^
 
-* The steps that constitute the progression bar are as follows:
+**Several simulation steps constitute the progression bar**
     #. Project Setup
     #. Inputs
     #. System Inputs
     #. Visualization
     #. Constraints
     #. Simulations
     #. Results
-* Each step has an associated index, with the progression being bar considered as an array
-* The current state of the view depends on the index of the current step
+**Each step has an associated index, with the progression being bar considered as an array**
```

Comment:
The attribute here is "a list of steps" with the property "Each step has an associated index, corresponding to its position in the list of steps"